docs/getting-started/quickstart-tt-debugging.md

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 3
+title: Time Travel Debugging Quickstart
+---
+
+Now that we've learned a little about programming DBOS, let's learn how to use the DBOS time travel debugger!
+
+This quickstart assumes you are using for [Visual Studio Code](https://code.visualstudio.com/) and have the 
+[DBOS Time Travel Debugger Extension](https://marketplace.visualstudio.com/items?itemName=dbos-inc.dbos-ttdbg) installed.
+
+:::info
+If you're not a VS Code user, please see the [Time Travel Debugging with DBOS CLI](../cloud-tutorials/timetravel-debugging#time-travel-with-dbos-sdk-cli-non-vs-code-users) tutorial.
+:::
+
+Additionally, this quickstart assume you have finished the [programming quickstart](./quickstart-programming).
```

Comment:
This is quickstart--Instead of making all these assumptions, I think we should provide concise directions to download it from GitHub, set it up, and deploy it to the cloud

Reply:
I thought it was better to build on the previous step rather than start from `create`, but I can definetly have the developer take out the 5 second wait

Reply:
I don't think we should assume people go through the tutorials in sequential order.

Reply:
The [programming quickstart](https://docs.dbos.dev/getting-started/quickstart-programming) "assumes you've finished our [quickstart](https://docs.dbos.dev/getting-started/quickstart)". Shouldn't we continue that in this quickstart?

Reply:
The programming quickstart also provides instructions on exactly how to set it up from scratch--I think we want the same thing here.

Reply:
It feels weird to repeat those instructions a third time. Please take a look at the latest revision and let me know if you think we need to repeat the `dbos-sdk init` instructions again

Reply:
It's better, but I still think it would be simpler and clearer if we made a new template and just said "run this command to get the code" instead of telling you to go to your old code (or a git repository) and edit it. Those additional steps, especially modifying the code, add needless friction.